source/stylesheets/components/_footer.scss

```diff
@@ -0,0 +1,116 @@
+//
+// Footer Component
+//
+// Based on the existing GOV.UK footer in GOV.UK Template, built with flexbox
```

Comment:
I think this comment might be out of date? I can't see any evidence of flexbox any more 🙂

Reply:
That's right. Will update the comment.